spring-xd-dirt/src/main/java/org/springframework/xd/dirt/container/store/ZooKeeperContainerRepository.java

```diff
@@ -153,6 +160,7 @@ private PathChildrenCache ensureCache() {
 						@Override
 						public void childEvent(CuratorFramework client, PathChildrenCacheEvent event) {
 							// shut down the cache if ZooKeeper connection goes away
+							ZooKeeperUtils.logCacheEvent(logger, event);
```

Comment:
Good idea! I've modified `ZooKeeperUtils.logCacheEvent` to log warnings for `CONNECTION_SUSPENDED` and `CONNECTION_LOST`.